src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/RunCodeActionService.cs

```diff
@@ -43,16 +44,19 @@ public RunCodeActionService(OmnisharpWorkspace workspace, [ImportMany] IEnumerab
             var operations = await action.GetOperationsAsync(CancellationToken.None);
 
             var solution = _workspace.CurrentSolution;
+            IEnumerable<OmniSharp.Models.ModifiedFileResponse> changes = Enumerable.Empty<OmniSharp.Models.ModifiedFileResponse>();
+            var directoryName = Path.GetDirectoryName(request.FileName);
             foreach (var o in operations)
             {
-                o.Apply(_workspace, CancellationToken.None);
+                var applyChangesOperation = o as ApplyChangesOperation;
```

Comment:
Does this mean that we will no longer run certain code actions that we still offer? I think that's the right thing since there are likely code actions that we're running with custom operations for displaying dialogs that can't be run. However, should there be a change elsewhere so that such code actions aren't offered?